plugins.txt

```diff
@@ -1,21 +1,22 @@
 docker-workflow:1.9.1
 ansible:0.6.2
-pipeline-milestone-step:1.2
```

Comment:
pipeline-milestone-step already exists in the base image now, as part of workflow-aggregator:

https://github.com/openshift/jenkins/blob/d0841a17f1e3ab64398c245042609dba327a1273/2/contrib/openshift/base-plugins.txt#L12..L15